docs_src/microservices/device/supported/device-usb-camera/supplementary-info/USB-protocol.md

```diff
@@ -0,0 +1,8 @@
+# USB Camera Device Service Specifications
+
+## USB Protocol Properties
+| Property | Description | EdgeX Value Type |               
+| -- | -- | -- |  
+| **Paths** | A list of internal [/dev/video paths](https://www.kernel.org/doc/html/v4.9/media/uapi/v4l/dev-capture.html) for the camera device. This list includes all streaming capable video paths for each device. | Object |  
+| **SerialNumber** | The serial number of the camera device. | String |  
+| **CardName** | The manufacturer name of the camera device. | String |
```

Comment:
Need to add `AutoStreaming` too.

Reply:
This section needs to be referred under Deployment section `Manage Devices` which describes Protocol properties of a USB device (list numbering needs a clean up too).